proto/panacea/datapool/v2/pool.proto

```diff
@@ -49,4 +49,9 @@ message DataAccessNFTRedeemReceipt {
   uint64 nft_id = 3;
   string redeemer = 4;      // 'panacea1' address of redeemer
   uint64 block_height = 5;  // The block height at which NFT redeemed
-}
\ No newline at end of file
+}
+
+// DataValidator defines a detail of data validator.
+message DataValidator {
+  string endpoint = 1;
+}
```

Comment:
I just put a `endpoint` field in `DataValidator` message. Refer to this [docs](https://github.com/medibloc/panacea-ecosystem-arch/blob/main/data-marketplace/data-pool-model/NFT-model.md#registering-a-data-validator-information), the some fields can be added.

Reply:
My guess is that the DataValidator should contain an address as well.
How about adding an address like this?
```
message DataValidator {
  string address = 1;
  string endpoint = 2;
}
```

Perhaps additional different values will be added during actual development.

Reply:
Yes, this might be better.